### DISCUSSION TITLE
How to set lsp signature help float window border.

### DISCUSSION THREAD

**Discussion by leisurelicht** [LazyVim/LazyVim#1320] [Q&A] [answered]
When I move Cursor in a function call, it will auto show a  float window. like this

<img width="967" alt="image" src="https://github.com/LazyVim/LazyVim/assets/8042345/3c540ed7-a9a5-4755-9a5f-8e412b074c82">

How I can set this float window's border to rounded  or disable this auto float window?

It's not vim.lsp.buf.signature_help. I already set signature help border like this

<img width="646" alt="image" src="https://github.com/LazyVim/LazyVim/assets/8042345/cd577888-504e-405c-94a3-505b0f5a6c07">
<img width="931" alt="image" src="https://github.com/LazyVim/LazyVim/assets/8042345/23bba567-56e4-4f5d-9593-c4d38339c148">

**Comment by dpetka2001** [comment]
I believe this comes from `nvim-cmp` itself. To add borders, see this [comment](https://github.com/LazyVim/LazyVim/discussions/419#discussioncomment-5802918) and check if it works for you.

**Reply by leisurelicht** [reply]
thanks. I add options from comment, but it not works . 
And I disabled `nvim-cmp` , still have this float window.
So I think  it's from `lsp-config`, but I not find any option to set borders in `lsp-config` doc.

**Reply by dpetka2001** [reply]
If it's definitely from `lsp-config` try the following
```lua
{
  "neovim/nvim-lspconfig",
  opts = {
    ui = {
      windows = {
        default_options = {
          border = "rounded",
        },
      },
    },
  },
}
```
When are you seing this window? Are you seeing it when you are writing something and waiting for auto-completions to appear? If so, it must be from `nvim-cmp` and the settings from the comment in the other thread should work, because I tested them out. Plz clarify, if you would, when you are seeing this float window, because it's hard for me to understand with certaintly just from the image.

**Reply by dpetka2001** [reply, marked as answer]
Ok I understand what you're saying. I believe this is what you want
```lua
  {
    "folke/noice.nvim",
    opts = {
      presets = {
        lsp_doc_border = true,
      },
    },
  },
```
Test it out to see if it's what you want.

**Reply by leisurelicht** [reply]
Yes, Thank you! Never thought it could be from noice.nvim.
